From 71d8f11060d265d525e7ad8b559d6dce772a6254 Mon Sep 17 00:00:00 2001
From: Dan Kephart <dan.kephart@lairdtech.com>
Date: Tue, 16 Feb 2016 17:42:41 -0500
Subject: [PATCH] laird: dts: wb50n: Remove interrupt configuration for
 ethernet

Remove the configuration of an interrupt on the WB50 for the ethernet mac.
The E_INT line coming off the 120-pin connector goes to a test point on the
BB40.  We need to be in polling mode because of this.

Bug: 8836
---
 arch/arm/boot/dts/wb50n.dts | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm/boot/dts/wb50n.dts b/arch/arm/boot/dts/wb50n.dts
index d6dda78..ecbc742 100644
--- a/arch/arm/boot/dts/wb50n.dts
+++ b/arch/arm/boot/dts/wb50n.dts
@@ -85,14 +85,6 @@
 			macb1: ethernet@f802c000 {
 				phy-mode = "rmii";
 				status = "okay";
-
-				#address-cells = <1>;
-				#size-cells = <0>;
-				phy0: ethernet-phy@1 {
-					interrupt-parent = <&pioE>;
-					interrupts = <30 IRQ_TYPE_EDGE_FALLING>;
-					reg = <1>;
-				};
 			};
 
 
-- 
1.9.1

